app/Checkout.php

```diff
@@ -50,4 +50,20 @@ public function kktSum(Semester $semester)
             ->where('semester_id', $semester->id)
             ->sum('amount');
     }
+
+    public function netregSum(Semester $semester)
```

Comment:
A helper function would reduce code dups here.

Reply:
I think it is enough if we add this when we will make the PaymentTypes dinamically added.

Reply:
we could still have the netregSum function but it could use a helper that does the same thing with PaymentTypes parameter